documentation/check_summary.sh

```
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+exclusions=('./SUMMARY.md' './release-notes/template' '.gitbook')
+allFiles=$(find . -name '*.md' | sort -d)
+missingFiles=()
+
+for file in $allFiles; do
+  missing=1
+  for exclusion in ${exclusions[@]}; do
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)